src/Kafka/ConsumerGroups/components/ConsumerGroupDetails/ConsumerGroupDetail.tsx

```diff
@@ -0,0 +1,23 @@
+import { FunctionComponent } from "react";
```

Comment:
Since this is the feature component it should live in the parent folder and have the same name as the finder, so `src/Kafka/ConsumerGroups/ConsumerGroup.tsx`. The exported component should also have the same name as the file.

Also, please flatten up the components folder, we don't need a folder per component.

This is described [in the docs](https://redhat-developer.github.io/app-services-ui-components/?path=/story/folder-structure-convention--page#how-to-structure-a-component)

Reply:
ConsumerGroupDetails is not the main feature, it's the internal component used to build the feature so I kept that in components folder